extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -0,0 +1,422 @@
+			CSSStyles: {
+				'display': 'flex',
+				'flex-direction': 'column',
+				'gap': '4px'
+			}
+		}).component();
+
+		const recommendedConfigurationLabel = view.modelBuilder.text().withProps({
+			value: constants.RECOMMENDED_CONFIGURATION.toUpperCase(),
+			description: "", // TODO - need this value later
+			height: 18,
+			CSSStyles: {
+				...styles.TOOLBAR_CSS,
+				'text-align': 'center',
+				'align': 'center',
+				'margin-left': '20px',
+			},
+		}).component();
+
+		this._recommendedConfigurationText = view.modelBuilder.text().withProps({
+			value: "",
+			height: 18,
+			CSSStyles: {
+				'font-size': '13px',
+				'line-height': '18px',
+				'font-weight': '600',
+				'margin': '0px',
+			},
+		}).component();
+
+		this._vmRecommendedConfigurationText = view.modelBuilder.text().withProps({
+			value: "",
+			height: 28,
+			CSSStyles: {
+				'font-size': '10px',
+				'line-height': '14px',
+				'font-weight': '400',
+				'margin': '0px',
+			},
+		}).component();
+
+		const viewDetailsLink = view.modelBuilder.hyperlink().withProps({
+			label: constants.VIEW_DETAILS,
+			url: '',
+			height: 18,
+			CSSStyles: {
+				'font-size': '13px',
+				'font-weight': '400',
+				'line-height': '18px',
+				'text-decoration': 'none',
+			}
+		}).component();
+
+
+		container.addItem(recommendedConfigurationLabel);
+		container.addItem(this._recommendedConfigurationText);
+
+		switch (this.migrationTargetType) {
+			case MigrationTargetType.SQLVM:
+				container.addItem(this._vmRecommendedConfigurationText);
+				container.addItem(viewDetailsLink);
+				break;
+			default:
+				container.addItem(viewDetailsLink);
+				container.addItem(this._vmRecommendedConfigurationText);
+		}
+
+		return container;
+	}
+
+	// TODO - We can remove dummayData input later and use a some other way to pass the values. Once we start implementing the whole page.
+	public async updateContent(dummyData: DummyData) {
+		await this._assessmentResultText.updateProperties({ "value": constants.ASSESSED_DBS(dummyData.assessmentResult) });
+		await this._readyText.updateProperties({ "value": dummyData.ready.toString() });
+		await this._needsReviewText.updateProperties({ "value": dummyData.needsReview.toString() });
+		await this._notReadyText.updateProperties({ "value": dummyData.notReady.toString() });
+		await this._blockersText.updateProperties({ "value": dummyData.blockers.toString() });
+		await this._warningsText.updateProperties({ "value": dummyData.warnings.toString() });
+
+		await this._recommendedConfigurationText.updateProperties({ "value": dummyData.skuRecommendation.toString() });
+		await this._vmRecommendedConfigurationText.updateProperties({ "value": dummyData.vmRecommendation });
+
+	}
+
+	// TODO - Check this later, if we need to handle this separately.
+	public dispose(): void {
+		this._disposables.forEach(
+			d => { try { d.dispose(); } catch { } });
+	}
+}
+
+export enum AssessmentResultType {
```

Comment:
Let's define this enum where all other enums are defined

Reply:
This enum is specific to this file. I saw similar for other files as well. Only enums common to different files are in utils.ts